platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -275,6 +275,8 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style;
 
+
+- (NSString *)mapViewMissingStyleImage:(NSString *)image;
```

Comment:
Something along the lines of:
`-(NSString *)mapView:(MGLMapView *)mapView didFailLoadingImage:(NSString *)missingImage;`

Reply:
`...didFailToLoadImage:` ?